test/provision/k8s_install.sh

```diff
@@ -77,6 +77,26 @@ networking:
 EOF
 )
 
+KUBEADM_CONFIG_ALPHA2=$(cat <<-EOF
+apiVersion: kubeadm.k8s.io/v1alpha2
+kind: MasterConfiguration
+api:
+  advertiseAddress: {{ .KUBEADM_ADDR }}
+  bindPort: 6443
+bootstrapTokens:
+- groups:
+  - system:bootstrappers:kubeadm:default-node-token
+  token: "{{ .TOKEN }}"
+kubernetesVersion: "v{{ .K8S_FULL_VERSION }}"
+networking:
+  dnsDomain: cluster.local
+  podSubnet: "{{ .KUBEADM_POD_NETWORK }}/{{ .KUBEADM_POD_CIDR}}"
+  serviceSubnet: 10.96.0.0/12
```

Comment:
Quick one, in case that we have IPv6, this still works? or need to have a different value?

Reply:
Ok, could you fix this now? Or add a note that need to be change?

Reply:
@eloycoto Fix what? Are we testing k8s with IPv6 in the CI?

Reply:
No, but there is a option for running test on IPv6, and If it's break I would like to have a note in there.

Reply:
Done